documentation/connect_puppet_master.markdown

```diff
@@ -16,77 +16,76 @@ canonical: "/puppetdb/latest/connect_puppet_master.html"
 [report_format]: ./api/wire_format/report_format_v5.html
 [puppetdb_server_urls]: ./puppetdb_connection.html#server_urls
 
-> Note: To use PuppetDB, your site's Puppet master(s) must be running Puppet 4.0.0 or later.
+> Note: To use PuppetDB, your site's Puppet master(s) must be running Puppet version 4.0.0 or later.
 
-After PuppetDB is installed and running, you should configure your Puppet master(s) to use it. Once connected to PuppetDB, Puppet masters will do the following:
+Once PuppetDB is installed and running, configure your Puppet master(s) to use it. When properly connected to PuppetDB, Puppet masters will do the following:
 
 * Send every node's catalog to PuppetDB
 * Send every node's facts to PuppetDB
 * Query PuppetDB when compiling node catalogs that collect [exported resources][exported]
 
-> Note: if you've [installed PuppetDB using the PuppetDB puppet module][install_via_module], then the `puppetdb::master::config` class is taking care of all of this for you.
+> Note: if you've [installed PuppetDB using the PuppetDB module][install_via_module], then the `puppetdb::master::config` class is taking care of all of this for you.
 
- **Working on your puppet master server(s),** follow all of the instructions below:
+ **Working on your Puppet master server(s),** follow all of the instructions below:
 
-## Step 1: Install Plugins
+## Step 1: Install plugins
 
-Currently, puppet masters need additional Ruby plugins in order to use PuppetDB. Unlike custom facts or functions, these cannot be loaded from a module and must be installed in Puppet's main source directory.
+Currently, Puppet masters need additional Ruby plug-ins in order to use PuppetDB. Unlike custom facts or functions, these cannot be loaded from a module and must be installed in Puppet's main source directory.
```

Comment:
plugin seems more correct to me, but we should be consistent throughout the docs at a minimum (see the header for instance)